lib/aamva/request/security_token_request.rb

```diff
@@ -37,8 +38,8 @@ def send
         raise ::Proofer::TimeoutError, message
       end
 
-      def self.auth_url
-        Env.fetch('AAMVA_AUTH_URL', DEFAULT_AUTH_URL)
+      def auth_url
+        config.auth_url || DEFAULT_AUTH_URL
```

Comment:
can we put the default where the config is loaded?

Reply:
Do you mean inline this method?

Reply:
sorry, misunderstood how it worked. this is good 🙂